README.md

```diff
@@ -32,7 +32,7 @@ If you ever want to update the stack:
 
 ```bash
 git pull
-docker-compose rm apiscraper
+docker-compose rm --force --stop apiscraper
```

Comment:
Just in case; this way you won't get prompted `[Yn]` to remove the old container, and it will stop it if it's running (say, you ran with `docker-compose up -d`)